scripts/index.js

```diff
@@ -25,7 +25,7 @@ const pokemonCard = (pokemon) => {
   div.appendChild(span)
 
   span.innerText = `#00${pokemon.id}`
```

Comment:
o que acontece se for um id com mais de dois digitos? ficaria, por exemplo, `#00137`? que tal fazer assim:
```js
const pokemonIndex = `00${pokemon.id}`
span.innerText = `#${pokemonIndex.slice(-3)}`
```